featurex/extractors/image.py

```diff
@@ -17,6 +17,15 @@
 except ImportError:
     pass
 
+try:
+    import pytesseract
```

Comment:
If the imported module is only called inside `apply`, we can probably move the import statement into the `apply` method. That way we also don't need to wrap it in a try/except block, since the tests won't run unless it's installed, and a user who tries to run it may as well see the exception so they know what's going on.